README.md

```diff
@@ -4,29 +4,69 @@ _Bulk import scripts for ingesting large external datasets into OWID's master da
 
 ## Development
 
-You should install Python 3.8+ and install required packages:
+1.  install Python 3.8+ and required packages:
 
 ```
 pip install -r requirements.txt
 ```
 
-Historical data transformations might have used Jupyter notebooks, but all recent ones use Python scripts.
+2. Copy `.env.example` to `.env` and change any variables as needed. If you are unsure what to change, ask a member of the OWID data team.
 
-## Updating data
+```
+cp .env.example .env
+```
+
+3. Follow the [setup instructions in the owid-grapher repository](https://github.com/owid/owid-grapher#initial-development-setup) to initialize a local version of the OWID MySQL database.
+
+### Folder structure
+
+Each dataset has its own folder `{institution}_{dataset}/` (e.g. `worldbank_wdi/` for the World Bank World Development Indicators), containing all code and configuration files required to execute the dataset import and suggest the chart revisions.
+
+Typical folder structure:
+
+```
+__init__.py # used for storing dataset constants (e.g. {DATASET_NAME}, {DATASET_VERSION})
+main.py     # executes all 6 steps in sequence
+...         # helper scripts (e.g. `download.py`, `clean.py`)
+input/      # original copy of the dataset
+output/     # the cleaned data to be imported + other *generated* files for steps 1-6
+config/     # *manually* constructed files for steps 1-6.
+```
+
+See [worldbank_wdi/](worldbank_wdi) for a recent example to follow.
+
+## Conventions to follow
+
+Each `{institution}_{dataset}/` folder executes the same 6 steps to import a dataset and suggest chart revisions:
+
+1. Download the data.
+   - Example: [worldbank_wdi/download.py](worldbank_wdi/download.py).
+
+2. Specify which variables in the dataset are to be cleaned and imported into the database.
+   - This information is typically stored in a `variables_to_clean.json` file. Example: [worldbank_wdi/output/variables_to_clean.json](worldbank_wdi/output/variables_to_clean.json). 
+   - For some datasets, it makes sense to generate `variables_to_clean.json` programmatically (as in [worldbank_wdi/init_variables_to_clean.py](worldbank_wdi/init_variables_to_clean.py)), in which case `variables_to_clean.json` should be stored in the `output/` sub-folder. For other datasets, it may make more sense for you to generate `variables_to_clean.json` manually, in which case it should be stored in the `config/` sub-folder.
+
+3. Clean the selected variables prior to import.
```

Comment:
I think maybe more detail here about what 'cleaning' the data entails and what the expected output .csvs are and what their expected formats are. I think just a copy or a reference to the README in the [standard importer repo](https://github.com/owid/importers/tree/master/standard_importer) would be good.